### PR TITLE
Restructure code files and smaller buttons in menu

### PR DESCRIPTION
**Pull request by babsey**
This PR restructures components of nodes and connections.

It adds model selection and parameter editing in menu.
I also reduced the size of the menu buttons in menu.

### CODE REVIEW

File: src/core/connection/connection.ts [hunk 1]
```diff
@@ -149,6 +150,19 @@ export class Connection extends Config {
     this.params.forEach((param: Parameter) => (param.visible = false));
   }
 
+  /**
+   * Sets default param value.
```
Comment (jens-br) [inline src/core/connection/connection.ts:154]:
```suggestion
   * Resets all parameters to their default value.
```

**Review by jens-br — CHANGES_REQUESTED** [review]
When I click on a node, I receive the error
```
vue.runtime.esm.js?2b0e:619 [Vue warn]: Unknown custom element: <NetworkNodeChip> - did you register the component correctly? For recursive components, make sure to provide the "name" option.

found in

---> <NetworkGraphToolbar> at src/components/network/NetworkGraphToolbar.vue
       <NetworkGraph> at src/components/network/NetworkGraph.vue
         <VMain>
           <Project> at src/views/Project.vue
             <VApp>
               <App> at src/App.vue
                 <Root>
```
I think we need to change in `NetworkGraphToolbar.vue` the lines  mentioned in the comment. An alternative would be to change ` NetworkNodeChip`  to export ` NodeChip` , if that was intended.

I think I liked "NetworkParamsEdit", since it indicated the editing of multiple parameters.

**Comment by babsey** [comment]
I changed the component name to NodeChip and updated the description for the function in connection.ts

**Review by jens-br — APPROVED** [review]
LGTM

**Pull request merged**